pkg/microservice/reaper/core/service/reaper/git.go

```diff
@@ -239,3 +273,37 @@ func (r *Reaper) buildGitCommands(repo *meta.Repo) []*c.Command {
 
 	return cmds
 }
+
+func writeSSHFile(sshKey, hostName string) error {
+	if sshKey == "" {
+		return nil
```

Comment:
If `sshKey` is empty, it may be better to consider it an error to call this function.